RadarSDK/Include/RadarTripOptions.h

```diff
@@ -25,7 +25,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (instancetype)initWithExternalId:(NSString *_Nonnull)externalId
             destinationGeofenceTag:(NSString *_Nullable)destinationGeofenceTag
      destinationGeofenceExternalId:(NSString *_Nullable)destinationGeofenceExternalId
-                scheduledArrivalAt:(NSDate *_Nullable)scheduledArrivalAt;
+                scheduledArrivalAt:(NSDate *_Nullable)scheduledArrivalAt
```

Comment:
This seems like a breaking change, would we need migration docs?

A follow up question is, do we want to keep on exposing these "init by fields" functions when we already expose the "init by dictionary" functions as we revamp the SDK for one line implementation? The upside would be that we will no longer be collecting deprecated interfaces. The downside would be that initializing from dictionary in native is kinda annoying and not developer friendly. However, one may also claim that with one line implementation all of these options that developers are currently building and loading into the client should be loaded in from the server.

Reply:
Correct, we shouldn't break this here. I'll either introduce another option with the 2 additional params or create a dictionary based initialization.